OracleDatabase/RAC/OracleDNSServer/dockerfiles/latest/Dockerfile

```diff
@@ -0,0 +1,80 @@
+# LICENSE UPL 1.0
+#
+# Copyright (c) 1982-2021 Oracle and/or its affiliates. All rights reserved.
+#
+# ORACLE DOCKERFILES PROJECT
+# --------------------------
+# This is the Dockerfile for Oracle Database 19c RAC Storage Server. This file create DNS Server for RAC. 
```

Comment:
Remove "Oracle Database 19c RAC Storage Server" and change the rest to "DNS server for RAC"